<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaletteSetup</class>
 <widget class="QDialog" name="PaletteSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>37</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Palette setup</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0,0,0">
   <item row="0" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QComboBox" name="cb_colors"/>
   </item>
   <item row="0" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="lbl_color">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>18</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_syntax">
     <property name="text">
      <string>Syntax element</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cb_syntax"/>
   </item>
   <item row="0" column="6">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cb_syntax</tabstop>
  <tabstop>cb_colors</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
